news/22.new.rst

```diff
@@ -0,0 +1 @@
+Add hook for ``AnyIO`` which dynamically imports its backend modules
```

Comment:
Can we have a `.` at the end of the sentence?

Reply:
Done.